### PR TITLE
German Translation for Copyright and Attributions

### PR DESCRIPTION
**Pull request by boris779**
### Description
German translation for copyright and attributions

### Motivation and Context
Missing german translation for that page

### Types of changes
- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist
- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/front_matter/copyright_and_attributions.de.md [hunk 1]
```diff
@@ -40,20 +34,20 @@ to the use of the information contained herein.
...
 {{% ghcontributors "https://api.github.com/repos/seleniumhq/www.seleniumhq.org/contributors?per_page=100" %}}
 
-#### [Previous Documentation Rewrite Project](//github.com/SeleniumHQ/docs/)
+#### [Vorgänger Projekt zur Neuentwicklung der Dokumentation](//github.com/SeleniumHQ/docs/)
 
 {{% ghcontributors "https://api.github.com/repos/seleniumhq/docs/contributors?per_page=100" %}}
 
 
-## Third-Party software used by Selenium documentation project:
+## rittanbieter Software die vom Selenium Dokumentationsprojekt genutzt wird:
```
Comment (diemol) [inline docs_source_files/content/front_matter/copyright_and_attributions.de.md]:
```suggestion
## Drittanbieter Software die vom Selenium Dokumentationsprojekt genutzt wird:
```

Comment (boris779) [reply docs_source_files/content/front_matter/copyright_and_attributions.de.md]: you are right.

Comment (diemol) [reply docs_source_files/content/front_matter/copyright_and_attributions.de.md]: 👍 if you want to, just click on `Commit suggestion` or push a new commit with the change, then we can merge it.

Comment (boris779) [reply docs_source_files/content/front_matter/copyright_and_attributions.de.md]: Done (with a push) changed your suggestion and did a merge from master

**Pull request merged**